app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikesController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    user = User.find(params[:user_id])
+    new_like = Like.new(author: user, post:)
```

Comment:
- [ ] The syntax error mentioned by the previous reviewer still exits. I suggest you rename the variable `post = Post.find(params[:post_id])` to something else maybe `user_post = Post.find(params[:post_id])` and use this variable on line 5. So it would be ` new_like = Like.new(author: user, post: user_post)`. This will clear the syntax error and also make your linters pass 👍. Also remember to change line 12 from `post.id` to `user_post.id` so that you don't encounter any error.

<img width="846" alt="chrome_KqRFwA7uSr" src="https://user-images.githubusercontent.com/81468675/155517667-63ca050b-a645-4c08-812f-ed53e5a488cc.png">